msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md

```diff
@@ -290,6 +299,90 @@ async def _mention_activity(self, turn_context: TurnContext):
 
 Now you can send an introduction message when your bot is first installed or added to a group or team.
 
+#### Tag mention
+
+Your bot can mention tags in text messages and Adaptive Cards posted into channels. When the bot @mentions the tag in a channel, the tag is highlighted and the people associated with the tag get notified. When a user hovers over the tag, a pop-up appears with the tag details.
+
+> [!NOTE]
+>
+> * Tag mentions are supported in Teams desktop and web clients.
+> * Tag mentions are supported in GCC and GCC-H tenants only.
```

Comment:
Tag mentions are supported in Public, GCC and GCCH.

Reply:
As discussed, we will not include public.